src/main/java/org/humanistika/oxygen/tei/completer/response/impl/JSONTransformer.java

```diff
@@ -25,6 +25,7 @@
 
 import static java.nio.charset.StandardCharsets.UTF_8;
 
+import java.nio.charset.StandardCharsets;
```

Comment:
This import is unneeded isn't it?

Reply:
Correct the IDE added it when i was going over the options